e2e/assets/pull/dfx.json

```
@@ -0,0 +1,31 @@
+{
+    "canisters": {
+        "e2e_project_backend": {
+            "type": "motoko",
+            "main": "src/e2e_project_backend/main.mo",
+            "dependencies": [
+                "dep1",
+                "dep2"
+            ],
+            "pull_ready": true,
+            "metadata": [
+                {
+                    "name": "dfx:wasm_url",
+                    "content": "example.com/e2e_project.wasm"
```

Should we be using urls with an `https://` or `http://` schema in the examples?
```suggestion
                    "content": "https://example.com/e2e_project.wasm"
```